run_tests.py

```diff
@@ -1,9 +1,23 @@
 # -*- coding: utf-8 -*-
 
-import unittest
+import os
 import sys
+import unittest
+
+def create_dir(path):
+    try:
+        os.mkdir(path)
+    except OSError:
+        print("%s already exists!" % path, file=sys.stderr)
+
+
+def set_up():
+    "Create files and directories for tests (e.g. exported data)"
+    create_dir(os.path.join(os.path.abspath('.'), 'tst', 'tmp'))
+    create_dir(os.path.join(os.path.abspath('.'), 'log'))
```

Comment:
:file_folder: `log/` keeps data from tests execution.
We could have :one: `Logger` class for the whole project or :two: `Logger`s:
* for main code
* and for tests.